server/src/app/controllers/userLogin.js

```diff
@@ -0,0 +1,27 @@
+import { User, bcrypt, jwt } from '../constants/packageConfig';
+import generateJwt from '../constants/generateJwt';
+
+const userLogin = async (req, res) => {
+  const { email, password } = req.body;
```

Comment:
Move all the controllers logic in try/catch so that you should not send the server errors on the front end.

Reply:
Done